packages/js/onboarding/changelog/53341-trunk

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: tweak
+
+ISO Code update for Odisha state (India) https://en.wikipedia.org/wiki/ISO_3166-2:IN (23 November 2023)
```

Comment:
@sorensd would you mind removing this changelog file? 🙏 somehow it got pushed but the correct one is the `plugins/woocommerce/changelog/53341-trunk`